src/pages/Admin/Venue/Details/VenueDetails.tsx

```diff
@@ -50,13 +50,13 @@ const VenueDetails: React.FC<VenueDetailsProps> = ({ venue }) => {
     id: venueId,
     rooms,
     mapBackgroundImageUrl = DEFAULT_MAP_BACKGROUND,
+    config,
   } = venue;
-  const {
-    subtitle,
-    description,
-    coverImageUrl,
-  } = venue.config.landingPageConfig;
-  const { icon } = venue.host;
+  const subtitle = config?.landingPageConfig?.subtitle;
+  const description = config?.landingPageConfig?.description;
+  const coverImageUrl = config?.landingPageConfig?.coverImageUrl;
```

Comment:
Why was this split into 3 seperate lines? Seemingly no need to extract `config` either?

```typescript
const {
    subtitle,
    description,
    coverImageUrl,
  } = venue.config?.landingPageConfig ?? {};
```

Reply:
Typescript doesn't allow object destructure from optional fields

Reply:
The code I posted above will absolutely work. The potentially undefined will get to be undefined, then the ?? {} defaults it to returning an object, which is then destructured. We already use this pattern across the codebase.

Reply:
Yeah, I didn't notice `?? {}` at the end. I tried without it and it doesn't work. But with this syntax seems to be accepted.

Reply:
Without it it’s trying to destructure undefined; with it it will try and destructure an empty object (which makes all the keys undefined)